analyzers/rspec/cs/S6803.html

```diff
@@ -1,4 +1,5 @@
-<p>Component parameters can only receive query parameter values in routable components with an @page directive.</p>
+<p>&lt;p&gt;This rule is deprecated, and will eventually be removed.&lt;/p&gt; Component parameters can only receive query parameter values in
```

Comment:
Is the format right? Do we want to keep the HTML tags ("&lt;p&gt;") in the description?

Reply:
Not sure - I'll update it anyways to be simple text.